types/gulp-html-prettify/gulp-html-prettify-tests.ts

```diff
@@ -0,0 +1,11 @@
+import * as gulp from 'gulp';
+import * as prettifyHtml from 'gulp-html-prettify';
```

Comment:
Since this is a callable or constructable entity, it isn't legal in ES2015 to import and call/construct this with a namespace import like `import * as foo from 'foo'`, so could you please convert this to either

```ts
import foo = require('foo')
```

or the following with `esModuleInterop` turned on

```ts
import foo from 'foo';
```

Reply:
Because of below error, I selected `import prettifyHtml = require('gulp-html-prettify');`.
```
Fix with a comment '// TypeScript Version: 2.7' just under the header.
Module '"C:/Users/***/Documents/WebStormProjects/other/DefinitelyTyped/types/gulp-html-prettify/index"' has no default export.
```